src/dotnet/Azure.ClientSdk.Analyzers/ci.yml

```diff
@@ -0,0 +1,27 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
This pipeline doesn't seem to be triggering.

Reply:
<img width="295" alt="rEQUIRE pr cOMMENTS" src="https://user-images.githubusercontent.com/31145988/131916344-cd6bd577-88ac-4331-8f11-98f752ab5d2f.PNG">
Its probably because this option is checked.

Reply:
@chidozieononiwu You didn't create a public pipeline for this ci.yml only an internal one. I renamed the old azure-sdk-tools - dotnet pipeline in public to tools - Azure.ClientSdk.Analyzers - ci to fix this. You might also need to setup a public pipeline for the snippet generator.